RELEASE_NOTES.md

```diff
@@ -152,6 +152,18 @@ bundle exec rake decidim:proposals:upgrade:remove_valuator_orphan_records
 
 You can see more details about this change on PR [\#10607](https://github.com/decidim/decidim/pull/10607)
 
+### 3.7. Initiatives pages exception fix
+
+We have addede a new tasks to fix a bug related to the pages component inside of the Initiatives module (`decidim-initiatives`).
+
+You can run the task with the following command:
+
+```console
```

Comment:
fo some reason, in my environment, this command is not being loaded. 
I have tried to Upgrade module, stop spring, use it via Rbenv, docker, this tasks is just not available.

Reply:
I've messed up in two ways:
First, the path, as I've added it on `decidim-initiatives/lib/tasks/upgrade/initiatives/` and it should be in `decidim-initiatives/lib/tasks/initiatives/upgrade/`
Second, the file extension, as it was `.rb` and it should be `.rake`
Both of these are fixed and now it should work correctly.

Reply:
I confirm that the page is being fixed after the task is being ran.